src/cookbook/navigation/set-up-universal-links.md

```diff
@@ -140,21 +138,21 @@ The hosted file should have the following content:
 }
 ```
 
-1. Set the `appID` value to your Flutter application ID.
+1. Set the `appID` value to `<team id>.<bundle id>`.
```

Comment:
I think this is fine, but maybe we could just link to and reference the above [App ID](https://docs.flutter.dev/cookbook/navigation/set-up-universal-links/#app-id) header since it gives more explicit details and avoids repeating ourselves.